mmf/models/base_model.py

```diff
@@ -174,13 +173,11 @@ def __call__(self, sample_list, *args, **kwargs):
         return model_output
 
     def load_requirements(self, *args, **kwargs):
-        if is_master():
-            requirements = self.config.get("zoo_requirements", [])
-            if isinstance(requirements, str):
-                requirements = [requirements]
-            for item in requirements:
-                download_pretrained_model(item, *args, **kwargs)
-        synchronize()
+        requirements = self.config.get("zoo_requirements", [])
+        if isinstance(requirements, str):
+            requirements = [requirements]
+        for item in requirements:
+            download_pretrained_model(item, *args, **kwargs)
```

Comment:
Did you test this after removing the is_master() check here? Different processes will try to download the resource tar.